flask_project/campaign_manager/static/js/detail/insight_functions.js

```diff
@@ -524,19 +547,20 @@ function renderFeatures(feature_type, feature_data, show_feature) {
             }
 
             var relationTag = capitalizeFirstLetter(relation['tags']['amenity']);
+            var completenessPercentage = (100 - relationTag['completeness']).toFixed(1);
 
             if(typeof relationTag !== 'undefined') {
                 L.polygon(latlngs, {
-                    color: '#' + intToRGB(hashCode(relationTag)),
-                    fillColor: '#' +  intToRGB(hashCode(relationTag)),
+                    color: colorCompleteness(relationTag['completeness']),
```

Comment:
hi @ann26 
i think from this code, it will be error, because relationTag is string
maybe should be relation['completeness']?